tests/download_data.py

```diff
@@ -74,7 +74,7 @@ def download():
     )
     print("Downloading unit_test_images")
     download_files(
-        "https://web.lcrc.anl.gov/public/e3sm/e3sm_diags_test_data",
+        "https://web.lcrc.anl.gov/public/e3sm/e3sm_diags_test_data/system_test/expected",
```

Comment:
I plan to do more refactoring in the next pull request, but I started implementing this before I realized how many more things should also be updated. Note that this directory name might change in the next pull request.